Speekins/stretch-project
# Feature/navbar css

#### What's this PR do?
This merge will add CSS for navbar including hover effects
#### Where should the reviewer start?
Run `git fetch` and `cd` into `feature/navbar-css`
#### How should this be manually tested?
In the root of the directory, run `npm start`. The webpage will be opened automatically.
- Navbar can be seen immediately on home page
- test out hover effects
- test out by clicking links to make sure the page is properly navigated to the correct url path
#### Any background context you want to provide?
#### Screenshots
![image](https://user-images.githubusercontent.com/74210902/211174695-60b04e7c-3ac9-491b-9565-3e1148bf6ae6.png)
